### DISCUSSION TITLE
Question, is tyhe OPTIONS stuff just for the Swagger doc?

### DISCUSSION THREAD

**Discussion by VictorioBerra** [Dotnet-Boxed/Templates#244] [Q&A]
https://github.com/Dotnet-Boxed/Templates/blob/master/Source/Content/ApiTemplate/Controllers/CarsController.cs#L24-L64

I assume this is built into MVC? Does it work with any route?

**Comment by RehanSaeed** [comment]
It's not for Swagger. It's to tell clients what HTTP methods are supported for a given resource. It's not strictly required to do it but it's a nice optional extra. REST...

Yes, it's built into MVC and you can create action methods for OPTIONS just like GET and PUT etc.
